packages/next/package.json

```diff
@@ -102,6 +102,7 @@
     "mini-css-extract-plugin": "0.8.0",
     "mkdirp": "0.5.1",
     "node-fetch": "2.6.0",
+    "object.assign": "4.1.0",
```

Comment:
You should use [`object-assign`](https://www.npmjs.com/package/object-assign) instead, as that is already a `react` dependency.